src/commands/lambda/__tests__/flare.test.ts

```diff
@@ -1,12 +1,64 @@
+import fs from 'fs'
+import process from 'process'
+import * as stream from 'stream'
+
+import axios from 'axios'
+import FormData from 'form-data'
+
+import {API_KEY_ENV_VAR, AWS_DEFAULT_REGION_ENV_VAR, CI_API_KEY_ENV_VAR} from '../constants'
+import {requestAWSCredentials} from '../prompt'
+
 import {createMockContext, makeCli} from './fixtures'
 
+const mockConfig = {
+  Environment: {
+    Variables: {
+      DD_API_KEY: 'some-api-key',
+      DD_SITE: 'datadoghq.com',
+      DD_LOG_LEVEL: 'debug',
+    },
+  },
+  FunctionArn: 'arn:aws:lambda:us-east-1:123456789012:function:some-function',
+  FunctionName: 'some-function',
+}
+
+jest.mock('../functions/commons', () => ({
+  getAWSCredentials: jest.fn(),
+  getLambdaFunctionConfig: jest.fn().mockImplementation(() => Promise.resolve(mockConfig)),
+  getRegion: jest.requireActual('../functions/commons').getRegion as () => string | undefined,
+}))
+
+jest.mock('../prompt', () => ({
+  requestAWSCredentials: jest.fn(),
+}))
```

Comment:
Wouldn't this do the same?
```suggestion
jest.mock('../prompt')
```